home/templates/home/blocks/highlighted_column_block.html

```diff
@@ -2,67 +2,58 @@
 
 <!-- single column template checks to ensure doesn't dispay additional columns -->
 {% if self.column_count == "12" %}
-  {% for item in self.column %}
-    {% if forloop.counter <= 2 %}
-      {% if item.highlight == "highlight" %}
-        <div class="m-cover m-highlight"></div>
-      {% elif item.highlight == "highlight-purple" %}
-        <div class="m-cover m-highlight-purple"></div>
-      {% else %}
-        <div class="m-cover">
-      {% endif %}
-        <div class="container">
-          <div class="row">
-          {% if item.highlight == "highlight" %}
-            <div class="block col-12 m-highlight">
-          {% elif item.highlight == "highlight-purple" %}
-            <div class="block col-12 m-highlight-purple">
-          {% else %}
-            <div class="block col-12">
+  {% for item in self.column|slice:":1" %}
+    {% if item.highlight == "highlight" %}
+      <div class="m-cover m-highlight"></div>
+    {% elif item.highlight == "highlight-purple" %}
+      <div class="m-cover m-highlight-purple"></div>
+    {% else %}
+      <div class="m-cover">
+    {% endif %}
+      <div class="container">
+        <div class="row">
+        {% if item.highlight == "highlight" %}
+          <div class="block col-12 m-highlight">
+        {% elif item.highlight == "highlight-purple" %}
+          <div class="block col-12 m-highlight-purple">
+        {% else %}
```

Comment:
You forgot the green here! And when I dont select a highlighting color, it doesnt show at all (only for 1 column). And, the link on a blue highlighted part becomes orange, is that how its supposed to be?

Reply:
hahahahs nope, will take a look :)